### ISSUE TITLE
`now dev` breaks when using TypeScript files within folders on Windows

### ISSUE THREAD

**Issue by tylerc** [zeit/now-cli#2678]
Title says it all. My app was working for me with `now dev` on Monday, but updates released since then broke any routes based on TypeScript files that are nested within folders.

For example, if `http://localhost:3000/api-broken` is configured to point to `zeit/api/api-handler.ts`, this is the output I get:

```
> GET /api-broken
> Building @now/node@canary:zeit/api/api-handler.ts
downloading user files...
download complete [1ms]
installing dependencies for user's code...
Found `engines` in `package.json`, selecting range: 10.x
installing to C:\Users\Tyler Church\Desktop\vanilla-from-wsl\zeit\api
npm WARN vanilla@1.0.0 No repository field.

audited 2 packages in 0.375s
found 0 vulnerabilities

install complete [4083ms]
running user script...
script complete [3ms]
tracing input files...
Using TypeScript 3.5.2 (now internal)
trace complete [1219ms]
> Built @now/node@canary:zeit/api/api-handler.ts [6s]
Cannot find module './zeit/api/api-handler.ts'
1) If you want to compile the package/file into executable, please pay attention to compilation warnings and specify a literal in 'require' call. 2) If you don't want to compile the package/file into executable and want to 'require' it from filesystem (likely plugin), specify an absolute path in 'require' call using process.cwd() or process.execPath.
Did you forget to add it to "dependencies" in `package.json`?
```

If I move the relevant `.ts` file to the root folder and update `now.json` to reflect this, things work. But obviously this is not ideal from a code organization standpoint.

The problem seems to have something to do with the built `.ts` file not getting copied or renamed or something. When the `.ts` file is in the root folder, the build output looks like this:

![image](https://user-images.githubusercontent.com/42798/62251801-c1911f80-b3a5-11e9-804a-522b9704842f.png)

But when I've got it in a folder, it looks like this:

![image](https://user-images.githubusercontent.com/42798/62251813-cc4bb480-b3a5-11e9-9935-d16450fb31ab.png)

Notice that the `.ts` file is missing in the second image, but a `.js` file is there in its place.

I'm running:

- Windows 10 v1903
- Node.js v10.16.0
- Now CLI v15.8.7
- NPM v6.9.0
- Yarn v1.17.3
- TypeScript v3.5.3

A repo that reproduces the issue is here: https://github.com/tylerc/zeit-now-ts-debugging

That app running is available here: https://zeit-now-ts-debugging.tylerc.now.sh

Luckily it seems to be working when deployed, but it makes development difficult on Windows machines.

I suspect that something related to this: https://github.com/zeit/now-builders/issues/855

I had to update my `now.json` to use `@now/node@canary` to fixed the error mentioned in that issue. But after doing so I ran into this new error.

If there's anything you'd like me to try, or any more information I can provide, please let me know.

**Comment by TooTallNate** [comment]
Thank you for the detailed repro steps @tylerc. We are looking into this.

**Comment by TooTallNate** [comment]
To help determine if this is related to `node-file-trace`, can you try it with `@now/node@0.11.1` and let us know if that works or not?

**Comment by tylerc** [comment]
@TooTallNate Yes! That does work! Switching to `@now/node@0.11.1` makes things work on both my test repo and in the app where I originally ran into this error.
